src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,118 @@
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and the co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to ensure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
+
+Requirements
+------------
+
+Create a Launch Checklist Form for astronauts to fill out in preparation for launch.
+The form should do the following:
+
+1. Validate the user responses with ``preventDefault()`` to ensure the following:
+
+   a. The user entered something for every field.
+   b. The user entered text for names and numbers for fuel and cargo levels.
+
+2. With validation, update a list of what is currently ready or not ready for the shuttle launch.
+3. Indicate what is good or bad about the shuttle and whether it is ready for launch by using the DOM to update the CSS.
+4. Fetch some planetary JSON to update the mission destination with vital facts and figures about where the shuttle is headed. 
+
+Setting Up Your Project Repository
+----------------------------------
+
+Fork the repository with the `starter code <https://github.com/LaunchCodeEducation/Launch-Checklist-Form/>`_ to your personal Github profile and clone the repository to the directory where you are keeping your assignments for the class.
+
+.. warning::
+
+   When updating styles to indicate whether the shuttle is ready for launch, do not modify ``styles.css``!
+
+To get started, navigate to the directory with your copy of the starter code. Open ``index.html`` with Firefox to verify that your starter code is working.
+
+When you open ``index.html``, you should see the Launch Checklist form with a rectangle above it for the mission destination and a rectangle below it that simply says "Awaiting Information Before Launch".
+
+.. figure:: figures/form-starting-point.png
+   :alt: Image showing the form and the box stating that more information is needed before launch.
+
+Adding Validation
+-----------------
+
+Adding Alerts
+^^^^^^^^^^^^^
+
+First, let's add the validation to notify the user if they forgot to enter a value for any one of the fields.
+
+This process is going to look similar to the :ref:`validation section <javascript-validation>` in the chapter on forms. 
+Make sure to use ``preventDefault()`` and an alert to notify the user that all fields are required.
+
+You also want to make sure that the user entered valid info for each of the fields.
+Valid information for the fields means that the user submits a value that is easily converted to the correct data type for our fellow engineers.
+The pilot and co-pilot names should be strings and the fuel level and cargo weight should be numbers.
+
+.. note:: 
+
+   If you want to check if something is ``NaN``, you cannot use ``==`` or ``===``.
```

Comment:
Slight rephrasing suggestion: "If you want to check if something is ``NaN``, you cannot use ``==`` or ``===``. Instead, JavaScript has a built-in method called ``isNaN(value)`` that returns ``true`` if ``value`` is ``NaN`` and ``false`` if ``value`` is not ``NaN``."

Reply:
Done! Thanks!